src/drafts.ts

```diff
@@ -11,3 +11,4 @@ export * from './Button2'
 export * from './ActionMenu2'
 export * from './DropdownMenu2'
 export * from './Label2'
+export * from './PageLayout'
```

Comment:
Should we directly put this in the main bundle with alpha maturity because this doesn't cause any breaking changes by being in the main bundle?

Reply:
Leaving this in drafts for now as we discussed in our engineering sync